### DISCUSSION TITLE
Help with Uploading GTFS - Pull Request approved

### DISCUSSION THREAD

**Discussion by GastonTransit** [transitland/transitland#441] [Q&A] [answered]
My GTFS is live on GoogleMap but after reading the documentation on TransitLand I am still having problems creating a DMFR file on GitHub. Any help is welcomed. 
GTFS: Gaston ACCESS, North Carolina
[export-2024-10-15T09-48-18.zip](https://github.com/user-attachments/files/17380720/export-2024-10-15T09-48-18.zip)

**Comment by drewda** [comment, marked as answer]
We've gone ahead and uploaded the file that you emailed to https://github.com/transitland/gtfs-archives-not-hosted-elsewhere/pull/140

And we've created a new DMFR file that points to that public location: https://github.com/transitland/transitland-atlas/pull/1264

It should be imported into the Transitland platform in the next few hours.

Whenever you have updates to the GTFS feed, feel free to open a PR against https://github.com/transitland/gtfs-archives-not-hosted-elsewhere or email us again at [info@interline.io](mailto:info@interline.io)
